### DISCUSSION TITLE
Ride-Hailing Q&A

### DISCUSSION THREAD

**Discussion by Lancy-Joy-Lobo** [ONDC-Official/mobility-specification#134] [Q&A]
Hi,

Is select call mandatory for auto/cab bookings, or is it possible to proceed with just search, Init and confirm calls without select? 
Additionally, will sending multiple confirm calls with the same transaction ID result in an error?

**Comment by 92shreyansh** [comment]
As per the current version that is released 2.0.0 select is a mandate, A good point here we can consider making this optional if the cohort agrees to it in the next 2.0.1 version

Also multiple confirm for same txn_id will result in an error
